R/utils.R

```diff
@@ -18,18 +18,16 @@ replace_null <- function(x, replacement) {
 #' Get unique vector values sorted by how often they occur
 #'
 #' @param x Vector, e.g. `c("a", "b", "b", "b", "a")`.
-#' @return Vector with unique values sorted by occurrence, e.g. `c("b", "a")`.
+#' @return Vector with unique values sorted by desc. count, e.g. `c("b", "a")`.
 #' @family helper functions
 #' @noRd
 unique_sorted <- function(x) {
-  dplyr::pull(
-    dplyr::arrange(
-      # Create data.frame with values ("Group.1") and how often they occur ("x")
-      stats::aggregate(x, by = list(x), FUN = length),
-      dplyr::desc(x)
-    ),
-    "Group.1"
-  )
+  # Create data frame with values and how often they occur
+  df <- stats::aggregate(x, by = list(x), FUN = length)
+  colnames(df) <- c("value", "count")
+  # Sort dataframe on count (can only be ascending) and retrieve values
+  values <- df[with(df, order(count)),][[1]]
+  rev(values) # Reverse order
```

Comment:
Maybe can you use arg `decreasing` in L29 and remove 30? To be tested...

Reply:
I have tried, that didn't work.